### ISSUE TITLE
No warning/error when importing a type only import without type annotation

### ISSUE THREAD

**Issue by itoxiq** [vue-vine/vue-vine#165]
When importing types without type annotation, it will throw an error.
There should be a warning when you are importing a type only import - i.E. importing Ref without type will not be highlighted in editor, but will throw an uncaught syntax error because the import will be invalid.

```typescript 
// example.vine.ts
// should error or throw
import { ref, Ref } from "vue";
```

Sidenote:
This would not throw an error when done inside a *.vue file.

**Comment by ShenQingchuan** [comment]
Good point you prompted to avoid this potential mistake.

That seems a lint issue to me.

I'll consider integrate this feature with @antfu/eslint-config's type aware rules to check.

**Comment by itoxiq** [comment]
I mean technically yes, but you get an error from vite when not using 'type' - where as in normal vue components, this does not happen (only linter complains)

**Comment by ShenQingchuan** [comment]
> I mean technically yes, but you get an error from vite when not using 'type' - where as in normal vue components, this does not happen (only linter complains)

Heard your reply! This problem is resolved in [**`v0.1.35`**](https://github.com/vue-vine/vue-vine/releases/tag/v0.1.35), please bump your dependency to try ~ 

Thanks for your kindly cooperation 💖
